### ISSUE TITLE
Support for Homebrew

### ISSUE THREAD

**Issue by ritvikgautam** [strumswell/BTTV-for-Safari#3]
Hey, 

I love your app, it's easy to use, and works just fine! 

I wanted to use homebrew to install this, instead of dealing with tar.gz's manually. 

I've written a homebrew cask cookbook that I'm going to be using to keep up with the updates, and just to manage it. 
For now it lives under my own tap name. I don't really care if you wanna pick it up and move it on your own so both the package and brew tap is from the same author. I'm happy as long as a homebrew solution is available. 

[Homebrew Cask Cookbook](https://github.com/ritvikgautam/homebrew-taps/blob/main/Formula/bttvforsafari.rb)
[Readme Updates/Instructions to Use](https://github.com/strumswell/BTTV-for-Safari/compare/main...ritvikgautam:BTTV-for-Safari:main)

Cheers!

**Comment by strumswell** [comment]
Good idea, didn't think about cask. I will provide support with the next update adding stuff from #2. Not sure when I have the time to do so tho, I guess within the next week. Thanks for providing guidance on that!
